pkg/netflow/flowaggregator/aggregator.go

```diff
@@ -137,3 +145,8 @@ func (agg *FlowAggregator) flush() int {
 
 	return len(flowsToFlush)
 }
+
+func (agg *FlowAggregator) rollupTrackersRefresh() {
+	log.Debugf("Rollup tracker refresh: use new store as current store")
+	agg.flowAcc.portRollup.UseNewStoreAsCurrentStore()
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
I know I know, I am usually the one in love with long names, but what about calling this just `Refresh` or `Reset` ?

Reply:
For this case, I think `Refresh` or `Reset` might not be suitable since since usually idempotent (can be call multiple time with same result), but in this `UseNewStoreAsCurrentStore` have a very precise operation that can't be called multiple times, and I would favour being explicit here.

Reply:
Wonder if the timer should be inside `portRollup`. You pass the observation window, and it will take care of dropping old data.

Reply:
I don't really agree with the last point :), I think it's good to keep the timer/scheduling part outside to keep `portRollup`. Usually, adding timing to portRollup makes it less reusable, reliant on time, and harder to test.